HedgeModManager/UI/MainWindow.xaml.cs

```diff
@@ -235,13 +295,34 @@ public void RefreshMods()
             }
         }
 
+        public void InvokeChangeCodesView(bool useTreeView = true)
+        {
+            if (!useTreeView)
+                goto InitListView;

```

Comment:
Try to avoid using goto's. Use methods to avoid repeating code. Define a method local to `InvokeChangeCodesView` and use that.